test/server/graphql/v2/collection/ExpenseCollectionQuery.test.ts

```diff
@@ -38,6 +38,16 @@ const expensesQuery = gqlV2/* GraphQL */ `
       customData: $customData
     ) {
       totalCount
+      aggregation {
+        currencyAmounts {
+          valueInCents
+          currency
+        }
+        totalAmount(currency: USD) {
+          valueInCents
+          currency
+        }
+      }
```

Comment:
Or:

```suggestion
      totalAmount(currency: USD)  {
        valueInCents
        currency
        amountsByCurrency {
          valueInCents
          currency
        }
      }
```

Reply:
I like this API; it's clear and simple.

If we stick with `aggregation`, I prefer `amountsByCurrency` as a name (rather than `currencyAmounts`). "Aggregation" has a benefit: we'll have a place to add more aggregations in the future, things like `expenses.aggregate.averageAmount`, `expenses.aggregate.medianAmount`, `expenses.aggregate.minAmount`, ...